pkg/tenant/db/members.go

```diff
@@ -14,23 +13,31 @@ import (
 	"github.com/vmihailenco/msgpack/v5"
 )
 
-const MembersNamespace = "members"
+const (
+	MembersNamespace       = "members"
+	MembersDefaultPageSize = 100
+)
 
 type Member struct {
-	OrgID        ulid.ULID `msgpack:"org_id"`
-	ID           ulid.ULID `msgpack:"id"`
-	Email        string    `msgpack:"email"`
-	Name         string    `msgpack:"name"`
-	Role         string    `msgpack:"role"`
-	Status       string    `msgpack:"status"`
-	Created      time.Time `msgpack:"created"`
-	Modified     time.Time `msgpack:"modified"`
-	DateAdded    time.Time `msgpack:"date_added"`
-	LastActivity time.Time `msgpack:"last_activity"`
+	OrgID        ulid.ULID    `msgpack:"org_id"`
+	ID           ulid.ULID    `msgpack:"id"`
+	Email        string       `msgpack:"email"`
+	Name         string       `msgpack:"name"`
+	Role         string       `msgpack:"role"`
+	Status       MemberStatus `msgpack:"status"`
+	Created      time.Time    `msgpack:"created"`
+	Modified     time.Time    `msgpack:"modified"`
+	DateAdded    time.Time    `msgpack:"date_added"`
+	LastActivity time.Time    `msgpack:"last_activity"`
 }
 
 type MemberStatus string
 
+const (
+	MemberStatusPending   MemberStatus = "Pending"
+	MemberStatusConfirmed MemberStatus = "Confirmed"
+)
```

Comment:
Could we create a 1 point story to follow on to this and convert MemberStatus to a uint8? That will dramatically reduce the size of the struct in the database. The enum can then have a string method that returns "pending" and "confirmed" for logging etc.

Reply:
Good idea, I'll create the story.